src/components/action-list/action-list.html

```diff
@@ -0,0 +1,160 @@
+<section class="docs-block">
+  <aside class="docs-block__info">
+    <h3 class="docs-block__header">Action list simple</h3>
+
+  </aside>
+  <div class="docs-block__content">
+    <div class="sg-actions-list sg-actions-list--space-between">
+      <div class="sg-actions-list__hole">
+        <button class="sg-button-secondary sg-button-secondary--small sg-button-secondary--inverse">
+          Answer
+        </button>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="docs-block">
+  <aside class="docs-block__info">
+    <h3 class="docs-block__header">Action list space between</h3>
+
+  </aside>
+  <div class="docs-block__content">
+    <div class="sg-actions-list sg-actions-list--no-wrap sg-actions-list--space-between-alt">
```

Comment:
`space-between-alt` doesn't say anything about it - what's different between this and the regular `space-between`?